### PR TITLE
feat: multiple updates for round 1 enhancements

### PR DESCRIPTION
**Pull request by sunnygleason**

### CODE REVIEW

**Comment by sunnygleason** [comment]
@mvines @CriesofCarrots ready for review!

* I'd like to do one round of React lint fixes after this is merged
* Then, I'll tackle the txn filter/bigger issue, I'd like to get these in first so that review is smaller

File: src/BxHelpLink.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ class BxHelpLink extends React.Component {
 
     return (
       <a
-        href={`https://solana-labs.github.io/book/terminology.html#${term}`}
+        href={`https://solana-labs.github.io/book-edge/terminology.html#${term}`}
```
Comment (mvines) [inline src/BxHelpLink.jsx]: Let's select "book" or "book-edge" based on the value of testnetDefaultChannel [1].  If testnetDefaultChannel `===` "edge", use "book-edge", else "book"


[1] https://github.com/solana-labs/solana-web3.js/blob/master/package.json#L27

File: src/App.js [hunk 2]
```diff
@@ -408,14 +419,15 @@ class App extends Component {
 
   handleSearch = self => event => {
     let value = event.target.value;
+    event.target.value = '';
 
     if (value.length > 80) {
-      event.target.value = '';
-      return self.handleClickOpen(value, 'txn')();
+        history.push(`/txn/${value}`);
```
Comment (CriesofCarrots) [inline src/App.js]: Incidentally, I agree with @mvines about spelling out "txn", "ent", etc. Also seems more RESTful.

File: src/BxHelpLink.jsx (hunk 1 above)
Comment (sunnygleason) [reply src/BxHelpLink.jsx]: @mvines ok, will mock something up. I'm a little fuzzy on how that will change/be overridden though (is it changed on deployment?)... isn't package.json immutable? In any event, patch coming...

File: src/App.js (hunk 2 above)
Comment (sunnygleason) [reply src/App.js]: OK, will ponder & follow up on channel!

**Pull request merged**